recap-loops/index.md

```diff
@@ -203,7 +228,10 @@ Output:
 2 x 10 = 20
 ```
 
-### Exercise 2: Dividable numbers
+### Graded: Dividable numbers
+
+This exercise is available in [Github classroom assignment here](https://classroom.github.com/a/KOXYDafF)
+
 - Read two numbers with Scanner
```

Comment:
This one's gonna confuse a bunch of them for sure, I didn't get it either until I saw the examples.
Maybe something like `Write a function that returns the largest element in an array of integer` might be more appropriate here?

Reply:
Yes, I agree that it is not the most intuitive. However I laid out already the skeleton in the classroom assignment so that it becomes more intuitive.  Let me see if I can improve a bit on the description anyway.